.expeditor/habitat_release.pipeline.yml

```diff
@@ -0,0 +1,25 @@
+expeditor:
+  defaults:
+    buildkite:
+      timeout_in_minutes: 30
+
+steps:
+#######################################################################
+# Release!
+#######################################################################
+
+  - label: "[lint] :linux: :bash: Shellcheck"
+    command:
+      - echo "HI"
+    agents:
+      queue: 'default-privileged'
+    plugins:
+      docker#v3.0.1:
+        always-pull: true
+        user: "buildkite-agent"
+        group: "buildkite-agent"
+        image: "chefes/buildkite"
+    timeout_in_minutes: 10
+    retry:
+      automatic:
+        limit: 1
```

Comment:
Any reason to retry Shellcheck, which should be deterministic?

Reply:
Oh I this is a placeholder (check the command). That being said any of the steps has the possibility of "infrastructure instability" so having a single retry in case of a blip is a good thing.